features/sentinel/repository/pom.xml

```diff
@@ -600,5 +601,12 @@
             <releases><enabled>false</enabled></releases>
             <snapshots><enabled>true</enabled></snapshots>
         </repository>
+        <repository>
```

Comment:
Do you need this ?

Reply:
Yes for sentinel container build for now.